### PR TITLE
cfscrape: init at 2.1.1

### PR DESCRIPTION
**Pull request by ghost**
###### Description of changes
https://github.com/Anorov/cloudflare-scrape

As suggested by @teto in #176331.
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] i686-linux
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by WeebSorceress** [comment]
Result of `nixpkgs-review pr 176416` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.cfscrape</li>
    <li>python39Packages.cfscrape</li>
  </ul>
</details>

File: pkgs/development/python-modules/cfscrape/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/Anorov/cloudflare-scrape";
+    description = "A Python module to bypass Cloudflare's anti-bot page";
+    license = licenses.mit;
+    platforms = [ "x86_64-linux" ];
```
Comment (teto) [inline pkgs/development/python-modules/cfscrape/default.nix:25]: i believe it's best to refer to platforms as the variables in lib/systems/doubles.nix, e.g. platforms.linux

**Comment by WeebSorceress** [comment]
Result of `nixpkgs-review pr 176416` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.cfscrape</li>
    <li>python39Packages.cfscrape</li>
  </ul>
</details>

**Comment by WeebSorceress** [comment]
Result of `nixpkgs-review pr 176416` run on x86-linux [1](https://github.com/Mic92/nixpkgs-review)

**Comment by WeebSorceress** [comment]
Result of `nixpkgs-review pr 176416` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.cfscrape</li>
    <li>python39Packages.cfscrape</li>
  </ul>
</details>

**Pull request merged**